cmd/apps/prometheus_operator.go

```diff
@@ -0,0 +1,142 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallPrometheusOperator() *cobra.Command {
+	var command = &cobra.Command{
+		Use:          "prometheus-operator",
+		Short:        "Install prometheus operator",
+		Long:         "Install prometheus operator",
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringArray("set", []string{},
```

Comment:
Define flags please, see the README.

https://github.com/helm/charts/tree/master/stable/prometheus-operator